doc/tutorial.rst

```diff
@@ -203,84 +199,9 @@ An example of a human-readable results file for our Titanic config file is::
     Additional Evaluation Metrics (Test):
      roc_auc = 0.8219429347826087
 
-
 IRIS Example on Binder
 ----------------------
 If you prefer using an interactive Jupyter notebook to learn about SKLL, you can do so by clicking the launch button below. 
 
 .. image:: https://mybinder.org/badge_logo.svg
  :target: https://mybinder.org/v2/gh/EducationalTestingService/skll/master?filepath=examples%2FTutorial.ipynb
-
-Running your own experiments
```

Comment:
Why was this section deleted? Is the jupyter notebook now the only way to learn about skll? If so, the comment above `If you prefer using` should be changed to something like `You can use`

Reply:
Sorry, this is not clear in the diff but much easier if you locally build the documentation and look at the TOC on the right. Essentially:

1) There are now two tutorials - one the regular Titanic one we have always had and the other is this new jupyter notebook we added.

2) The "Running your own experiments" section didn't really belong in the tutorial section so I called it "General Workflow" and moved it to be the top sub-section under the "Running experiments" section.

Reply:
Ah, sorry, yes I see that now. Somehow I missed that whole file.